propresenter_bible/install/installer.py

```diff
@@ -0,0 +1,395 @@
+        rvmetadata = folder / "rvmetadata.xml"
+        if not rvmetadata.is_file():
+            raise FileNotFoundError(f"rvmetadata.xml not found for folder {folder_id}")
+        xtree = ElementTree.parse(str(rvmetadata))
+        abbr_tag = xtree.xpath("//abbreviation")[0]
+        abbr_tag.text = new_internal_abbr
+        rvmetadata.write_bytes(toxml(xtree))
+
+        # update entry and save
+        for e in entries:
+            if e.folder_id == folder_id:
+                e.abbreviation = new_internal_abbr
+                break
+        self._save_entries(entries)
+
+    def get_overwrite_info(self) -> dict:
+        info = {}
+        root = self.get_bibles_root_dir()
+        if not root:
+            return info
+        try:
+            installed_raw = self.read_installed_bibledata()
+            for raw in installed_raw:
+                meta = self.parse_bible_meta(raw)
+                folder = root / meta["id"]
+                # Determine rightsHolderAbbreviation
+                rights = None
+                installed_name = None
+                try:
+                    mpath = folder / 'metadata.xml'
+                    if mpath.is_file():
+                        xt = ElementTree.parse(str(mpath))
+                        node = xt.xpath('//DBLMetadata/contact/rightsHolderAbbreviation')
+                        rights = node[0].text if node else None
+                        # Try to get installed name (prefer local name if you want)
+                        n = xt.xpath('//identification/name')
+                        if n:
+                            installed_name = n[0].text
+                        else:
+                            nloc = xt.xpath('//identification/nameLocal')
+                            installed_name = nloc[0].text if nloc else None
+                except Exception:
+                    rights = None
+                # displayAbbreviation from rvmetadata.xml
+                display = None
+                try:
+                    rv = folder / 'rvmetadata.xml'
+                    if rv.is_file():
+                        xt = ElementTree.parse(str(rv))
+                        node = xt.xpath('//displayAbbreviation')
+                        display = node[0].text if node else None
+                except Exception:
+                    display = None
+                status = 'Overridden' if rights == 'NBV21' else 'Original'
+                info[meta["abbreviation"].lower()] = {
+                    'status': status,
+                    'location': str(folder),
+                    'displayAbbreviation': display,
+                    'name': installed_name,
+                }
+        except Exception:
+            return info
+        return info
+
+
+class MacInstaller(InstallerBase):
+    """Installer implementation for macOS (Darwin)."""
+
+    def get_sideload_dir(self) -> Path:
+        return Path('/Library/Application Support/RenewedVision/RVBibles/v2/')
```

Comment:
@martijnlentink Based on my testing this path needs to be in the user directory, at least for ProPresenter 20+

```suggestion
        import os
        return Path(os.path.expanduser('~')) / 'Library/Application Support/RenewedVision/RVBibles/v2/'
```

Reply:
Thank you for taking the time to test this. I've had messages from people telling me that installing bibles on OSX didn't work properly with the latest versions of ProPresenter/OSX. Do you think this would solve it? How did you experiences issues with the latest release (main branch) as well?

Reply:
Yes, the same issue exists on the `main` branch (the .rvbible files end up in a different folder than where Pro7 looks).
I assume the behaviour change was on the ProPresenter side, but I haven't tested any older version of Pro7 to confirm.

Reply:
That is amazing, thanks for that! That will help out a lot of people who run this on their Mac. I'll change this in the main branch as well and draft a new release ASAP to bugfix it. While I have you, did you miss anything with this UI version, did it feel intuitive for you?

Reply:
<img width="1007" height="333" alt="Screenshot 2025-10-02 at 7 01 21 pm" src="https://github.com/user-attachments/assets/83907fc1-3f8c-49f2-b903-b2b7dab412e4" />

It's definitely intuitive, just that the ProPresenter tab doesn't render correctly on MacOS with dark mode enabled.

Reply:
Thanks, that is a good point!